batch/batch/front_end/front_end.py

```diff
@@ -826,7 +829,7 @@ async def _create_jobs(userdata: dict, job_specs: dict, batch_id: int, app: aioh
                 if spec['process']['type'] == 'jvm':
                     if 'cpu' in resources:
                         raise web.HTTPBadRequest(reason='jvm jobs may not specify cpu')
-                    if 'memory' in resources:
+                    if 'memory' in resources and resources['memory'] != 'standard':
```

Comment:
I'm not sure what happened with this change? Is this something I added? Maybe Dan removed it already from main?

Reply:
I'm not sure if it was you or Dan who changed it. Should I keep it as is?